src/execute/index.js

```diff
@@ -76,7 +75,14 @@ export function execute({
     ...extras,
   });
 
-  if (request.body && (isPlainObject(request.body) || Array.isArray(request.body))) {
```

Comment:
Comparing this change with original, I'd say we'll stick with `isPlainObject` for the time being. Determining if something is POJO is a hard job, and having a well tested utility function that abstract the complexity just make sense. 

I think we have two options with `isPlainObject`
1.) remove the `isPlainObject` from the list and keep using lodash version
2.) create our own utility function that does example the same as `isPlainObject` from lodash


The scope of #2187 (as I understood it) is to replace lodash functions with modern & native simple javascript constructs. IMHO this doesn't fall within that scope. 

// cc @jimmywarting

Reply:
What @char0n have mention about `isPlainObject` not being equivalent to just a simple `typeof` is true... need to check more things such as function, null etc

in some senarios it's fine to use just `typeof x === object` but we need to know for sure if it's okey to use it.

the point of my issue is not to use any lodash at all, even if you just include one simple function then lodash will at least load some internal core functions that are pretty big in itself and is deeply dependent on many other stuff to make it compatible with es3 or 5 (whatever it is).

Right now there is so few lodash fn that we use that i don't see any point in keeping it, so i suggest # 2 
`create our own utility function that does the same thing`

Reply:
where can i  create function isPlainObject? In which project folder?

Reply:
@gabrielsimongianotti `src/helpers.js` contains already two helper functions at the top of the file. Le'ts put `isPlainObject`

Reply:
Another options is to use https://www.npmjs.com/package/is-plain-object

Contains the logic that we need and handles corner cases. Has no dependencies as well.